api/client/samples/crop_w_series/soybean_w_drought_index.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>It's not quite consistent with the previous plot.</p><p>Could you also set <code>legend=True</code>  here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>